common/crypto_utils/avalon_crypto_utils/keys.py

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 import hashlib
-import avalon_crypto_utils.crypto.crypto as crypto
+from ecdsa import VerifyingKey
+from Cryptodome.PublicKey import RSA
```

Comment:
Enclave Manager this not using the EnclaveKeys. Object  is getting initialzied but seems it's not used. For example 
https://github.com/hyperledger/avalon/blob/master/enclave_manager/avalon_enclave_manager/singleton/singleton_enclave_info.py#L61

You could remove the occurrence of `EnclaveKeys` and also remove this file `keys.py`

Reply:
Done